q2_amr/amrfinderplus/types/_format.py

```diff
@@ -0,0 +1,67 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2019-2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from q2_types.feature_data import MixedCaseDNAFASTAFormat, ProteinFASTAFormat
+from qiime2.plugin import model
+
+
+class TextFormat(model.TextFileFormat):
+    def _validate_(self, level):
+        pass
+
+
+class BinaryFormat(model.BinaryFileFormat):
+    def _validate_(self, level):
+        pass
+
+
+class AMRFinderPlusDatabaseDirFmt(model.DirectoryFormat):
+    amr_lib = model.File("AMR.LIB", format=TextFormat)
+    amr_lib_comp = model.FileCollection(r"^AMR\.LIB\.h3.$", format=BinaryFormat)
+    amrprot = model.File("AMRProt", format=ProteinFASTAFormat)
+    amrprot_blast = model.FileCollection(r"^AMRProt\.p..$", format=BinaryFormat)
+    amrprot_mutation = model.File("AMRProt-mutation.tab", format=TextFormat)
+    amrprot_suppress = model.File("AMRProt-suppress", format=TextFormat)
+    amrprot_susceptible = model.File("AMRProt-susceptible.tab", format=TextFormat)
+    changes = model.File("changes.txt", format=TextFormat)
+    db_version = model.File("database_format_version.txt", format=TextFormat)
+    fam = model.File("fam.tab", format=TextFormat)
+    taxgroup = model.File("taxgroup.tab", format=TextFormat)
+    version = model.File("version.txt", format=TextFormat)
```

Comment:
I'm wondering here: what's the difference between the contents of `db_version` and `version`? Also is there anything interesting in the `changes`? Technically, we would not need to maintain all of these files. Maybe just a single file with version would be enough?

Reply:
i just thought it would be easier to leave everything in because the files are very small anyway. But I can also take them out.
actually the AMR_CDS files are also not needed, so I'll remove them too.

Reply:
Ah yes i forgot to change it in the PR test files but yes that is how it should be. :) 
When you approve, can you also merge for me. Because those required checks are still not gone.